models/marts/fct_order_items.sql

```diff
@@ -0,0 +1,59 @@
+with order_items as  (
+    select * from {{ ref('stg_order_items' )}}
+),
+
+orders as  (
+    select * from {{ ref('stg_orders' )}}
+),
+
+order_payments as  (
+    select * from {{ ref('stg_order_payments' )}}
+),
+
+reviews as (
+    select * from {{ ref('stg_order_reviews') }}
+),
+
+final as (
+
+    select
+    -- primary key
+        order_items.order_item_sk,
+    -- foreign keys
+        order_items.order_id,
+        order_items.product_id,
+        order_items.seller_id,
+        orders.customer_id,
+    -- timestamps
+        orders.order_purchase_timestamp as ordered_at,
```

Comment:
this type of renaming should happen in the stage model

Reply:
noted, updated!